### PR TITLE
Static Code Eslint

### PR DESCRIPTION
**Pull request by SebastianGomezVidal**
image

### CODE REVIEW

File: .eslintrc.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  node: true
+  mocha: true
+extends:
+  - eslint:recommended
+  - airbnb-base
+  - plugin:import/errors
+  - plugin:import/warnings
+rules:
+  "comma-dangle": ["error", "never"]
+
```
Comment (holgiosalos) [inline .eslintrc.yml:12]: Line 12 has whitespaces, remove them

File: .eslintrc.yml [hunk 2]
```diff
@@ -8,5 +8,4 @@ extends:
   - plugin:import/errors
   - plugin:import/warnings
 rules:
-  "comma-dangle": ["error", "never"]
-  
\ No newline at end of file
+  "comma-dangle": ["error", "never"]
```
Comment (holgiosalos) [inline .eslintrc.yml:11]: EOF

File: .eslintrc.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  es6: true
+  node: true
+  mocha: true
+extends:
+  - eslint:recommended
+  - airbnb-base
+  - plugin:import/errors
+  - plugin:import/warnings
+rules:
+  "comma-dangle": ["error", "never"]
```
Comment (kliver98) [inline .eslintrc.yml:11]: add an enter, in total should be 12 lines

Comment (SebastianGomezVidal) [reply .eslintrc.yml]: Done

**Review by kliver98 — APPROVED** [review]
Looks good to me

**Pull request merged**